CMakeLists.txt

```diff
@@ -47,6 +47,67 @@ if(POLICY CMP0135)
   cmake_policy(SET CMP0135 NEW)
 endif()
 
+
+if(WIN32)
+    if(DEFINED ENV{BUILDENV_BASEPATH})
+        set(BUILDENV_BASEPATH "$ENV{BUILDENV_BASEPATH}\\")
+    else(NOT DEFINED BUILDENV_BASEPATH)
+        set(BUILDENV_BASEPATH "${CMAKE_SOURCE_DIR}/buildenv/")
```

Comment:
I think this is redundant with MIXXX_VCPKG_ROOT.

Reply:
I wonder how it is done in case of git worktrees. I can imagine to have only one buildenv folder that is accessed by all worktrees. Did you consider that? That will safe a lot HD and bandwidth.

Reply:
Neither the variable nor the directory structure is changed, in this PR. This is therefore out of scope.
But I think it should help in this usecase, that the variable is now an environment variable, instead of a batch file internal variable. This allows the user to overwrite it after executing the script.

Reply:
I will replace MIXXX_VCPKG_ROOT with BUILDENV_BASEPATH, than the variable naming scheme becomes consistent.

Reply:
It depends. I have no special demands here, just note that the VCPKG_ROOT env variable is the system wide default. We have created our own derived from that name to be independent from it.

Reply:
Looking at this again, it's `BUILDENV_PATH` which is redundant to `MIXXX_VCPKG_ROOT` and not `BUILDENV_BASEPATH`.